### PR TITLE
Fix build issue.

### PR DESCRIPTION
**Pull request by ichergui**
Alignment of git submodules in dunfell-l4t-r32.4.3 branch.

### CODE REVIEW

**Comment by ichergui** [comment]
Build issue when trying  the following command:

File: .gitmodules [hunk 1]
```diff
@@ -17,7 +17,7 @@
 [submodule "repos/meta-mender-community"]
 	path = repos/meta-mender-community
 	url = https://github.com/OE4T/meta-mender-community.git
```
Comment (dwalkes) [inline .gitmodules:19]: @ichergui thanks for the contribution.

@madisongh @kekiefer I think based on the conversation at https://github.com/OE4T/tegra-demo-distro/pull/1#issuecomment-693445349 the plan was to point meta-mender-community dunfell at the upstream mender repo?

Comment (dwalkes) [reply .gitmodules:19]: I've verified the build succeeds with https://github.com/BoulderAI/tegra-demo-distro/tree/dunfell-mender-upstream and https://github.com/BoulderAI/tegra-demo-distro/commit/142df3dfc2ef8b616478bf576185f2f03258c8a7 using `jetson-tx2` although I do see patch fuzz on the u-boot patches.  We'll get a PR into meta-meta-mender-community dunfell branch this week to fix the patch fuzz.

Comment (ichergui) [reply .gitmodules:19]: Hi @dwalkes 
Do you want me to update my PR ? Or maybe we should wait for Matt thoughts about this.

Comment (dwalkes) [reply .gitmodules:19]: We can let @madisongh or @kekiefer weigh in, I might not have understood the plan here, or maybe we should wait until we fix the patch fuzz.

Comment (madisongh) [reply .gitmodules:19]: We need to update both meta-mender and meta-mender-community to dunfell.  I've got that setup in my fork and see no patch fuzz for u-boot.

Comment (ichergui) [reply .gitmodules:19]: Thanks @madisongh for your response.

Comment (madisongh) [reply .gitmodules:19]: @ichergui I've got your two changes (slightly modified) along with some other fixes I'm pulling in from my [fork](https://github.com/madisongh/tegra-test-distro) over at PR #7.  I'll run a test build or two and get them merged assuming they look OK.

Comment (ichergui) [reply .gitmodules:19]: Perfect.
I can do the same test as you are doing.
What do you think ?

Comment (madisongh) [reply .gitmodules:19]: Sure, if you've got the time, that would be great.  My first build is still in progress, almost done.

Comment (ichergui) [reply .gitmodules:19]: Build done successfully.
I will do a quick test.

Comment (madisongh) [reply .gitmodules:19]: Thanks @ichergui ... I found a couple of issues and am fixing them up now.  Will refresh PR #7 shortly.

Comment (ichergui) [reply .gitmodules:19]: Yeah, no problem with a pleasure.
I flashed my device `jetson-tx2`.
Here is the result of some commands: 
* `$ cat /etc/os-release `
*  `# mender show-artifact`
```
root@jetson-tx2:~# cat /etc/os-release 
ID=tegrademo-mender
NAME="OE4Tegra Demonstration Distro with Mender"
VERSION="3.1--ichergui"
VERSION_ID="3.1--ichergui"
PRETTY_NAME="OE4Tegra Demonstration Distro with Mender 3.1--ichergui"
root@jetson-tx2:~# mender show-artifact
demo-image-base_3.1--ichergui
root@jetson-tx2:~# 
``` 

```
$ bitbake -e demo-image-base | grep -i "^BUILDNAME"
BUILDNAME="3.1--ichergui"
$ 
```

I think we should made a small update [here](https://github.com/OE4T/tegra-demo-distro/pull/7/commits/17039b41a876fc2c6890d5653bb66d82895ead7d#diff-1b6b714395cc684c80eceb7bf9ab813e02797c9d1baab9cd3b1fb9f5590b4ff3R15)

What do you think ?

Comment (madisongh) [reply .gitmodules:19]: Yeah, I saw that and have a fix for it... you should see it in my next push for #7

Comment (ichergui) [reply .gitmodules:19]: OK, Perfect.
Thanks @madisongh

**Comment by madisongh** [comment]
Merged with #7

**Pull request closed without merging**